api/config/config.defaults.ts

```diff
@@ -89,7 +89,7 @@ const config: DeepPartial<Config> = {
         options: {
           name: 'twine-api-session',
           cookieOptions: {
-            password: 'pefwj48208tfh29pd9cm10cprmhx94pfry28ctnc2mkrx0dufpog3',
```

Comment:
Just checking this does not need to be removed from git history

Reply:
I think it's fine, it was just me mashing the keyboard for a temp password to use while developing. 

I've not used the same password for the environment variable in the `.env` file locally and wont use the same one in the env vars on the hosted environments.